src/System.IO.Compression/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -72,7 +72,7 @@ public enum CompressionLevel : int
         {
             NoCompression = 0,
             BestSpeed = 1,
-            BestCompression = 9,    // Refer to "How to choose a compression level" above.
+            BestCompression = 6,    // Refer to "How to choose a compression level" above.
```

Comment:
I don't think this is the correct change.  Zlib's "BestCompression" value is 9, so if we have an enum value here called "BestCompression", its value should match zlib's value.